src/Extend/Frontend.php

```diff
@@ -185,4 +183,26 @@ private function getModuleName(?Extension $extension): string
     {
         return $extension ? $extension->getId() : 'site-custom';
     }
+
+    public function onEnable(Container $container, Extension $extension)
+    {
+        if (! empty($this->js) || ! empty($this->css)) {
+            $recompile = new RecompileFrontendAssets(
+                $container->make('flarum.assets.'.$this->frontend),
+                $container->make(LocaleManager::class)
+            );
+            $recompile->flush();
```

Comment:
Could we extract this block into a private method on the extender, please? It's now duplicated very often in this file.